Pico8Emulator/unit/Unit.cs

```diff
@@ -23,5 +23,10 @@ public class Unit {
 		public virtual void Update() {
 			
 		}
+
+        public virtual void OnCartridgeLoad()
+        {
+
+        }
```

Comment:
My tabs insert 4 spaces. I think it's better this way since every environment interprets tabs their own way. This way it'll be 2/4 spaces across the board!